btb/tuning/hyperparams/boolean.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+"""Package where the boolean hyperparameters are defined."""
+
+import numpy as np
+
+from btb.tuning.hyperparams.base import BaseHyperParam
+
+
+class BooleanHyperParam(BaseHyperParam):
+    """Boolean Hyperparameter Class.
+
+    The boolean hyperparameter class it is responsible for the transformation of boolean values in
+    to normalized search space of [0, 1]^K and also providing samples of those or the inverse
+    transformation from search space to hyperparameter space.
+
+    Hyperparameter space:
+        ``{True, False}``
+
+    Search space:
+        ``{0, 1}^K``
+
+    Attributes:
```

Comment:
This can be removed from here, as this K is actually defined and documented in the BaseHyperParam